### PR TITLE
fix: Fix issues in manifests apps repository

### PR DESCRIPTION
**Pull request by mimotej**
This PR fixes various issues which were reported by kubeval tool. This changes were made using updated schemas which are not yet merged: https://github.com/operate-first/schema-store/pull/7
Part of: https://github.com/open-services-group/scrum/issues/6

### CODE REVIEW

File: cluster-scope/overlays/prod/emea/morty/ingresscontrollers/default.yaml [hunk 1]
```diff
@@ -4,13 +4,7 @@ metadata:
   name: default
   namespace: openshift-ingress-operator
 spec:
-  clientTLS:
-    clientCA:
-      name: ""
-    clientCertificatePolicy: ""
   httpEmptyRequestsPolicy: Respond
-  httpErrorCodePages:
-    name: ""
```
Comment (tumido) [inline cluster-scope/overlays/prod/emea/morty/ingresscontrollers/default.yaml:13]: These are valid options in OCP 4.9, removing them will change the cluster behavior.

https://docs.openshift.com/container-platform/4.9/networking/ingress-operator.html

**Review by tumido — CHANGES_REQUESTED** [review]

File: cluster-scope/overlays/prod/emea/morty/ingresscontrollers/default.yaml (hunk 1 above)
Comment (mimotej) [reply cluster-scope/overlays/prod/emea/morty/ingresscontrollers/default.yaml]: Resolved in latest commit

**Review by tumido — COMMENTED** [review]
/lgtm

**Comment by larsks** [comment]
/lgtm

Although I really wish the commit message had some content (such as the pr description).

**Pull request merged**